client/informant/src/lib.rs

```diff
@@ -29,18 +29,22 @@ use std::time::Duration;
 mod display;
 
 /// The format to print telemetry output in.
-#[derive(PartialEq)]
+#[derive(Clone)]
 pub enum OutputFormat {
-	Coloured,
-	Plain,
+	Coloured {
+		prefix: String,
```

Comment:
Why is the prefix part of these variants?

Reply:
It makes sense to me.

Maybe it would make more sense to have this:
```rust
struct Format {
    color: bool,
    prefix: String,
}
```

But I noticed that we don't really need the distinction between coloured and plain at all because the colors are strips anyway. (As I said here: https://github.com/paritytech/substrate/pull/6174#discussion_r431832703 )

I think we should delete it and keep only the prefix.

Reply:
Sounds like color stripping is a bug?

The new struct looks more reasonable.

Reply:
Ok for the new struct.

> Sounds like color stripping is a bug?

No I think it is made on purpose: https://github.com/paritytech/substrate/commit/f77b3e3206c81ce5deb1e78c95302fc209a88526#diff-4b08e5c0745ff570ad2a3be5fdaf2b5aR477

Plus it kinda makes sense to format everything with colors and strip them automatically so you don't have to maintain 2 pathways.

@arkpar do you have any comment on this?

Reply:
Color is stripped to make sure it is not written to non-terminal outputs, such as file. You can have the same log string be printed in the console with color, and duplicated to stderr with color stripped.